src/Microsoft.Azure.WebJobs.Logging/README.md

```diff
@@ -0,0 +1,9 @@
+﻿# Microsoft.Azure.WebJobs.Host.Logging
```

Comment:
This Logging package doesn't seem to be used by anyone anymore. It's the old Cosmos Table logging functionality.